lib/puppet/provider/augeas/augeas.rb

```diff
@@ -203,7 +203,7 @@ def close_augeas
 
   def is_numeric?(s)
     case s
-    when Fixnum
+    when Integer,Float
```

Comment:
If also attempting to fix the potential bug, I would like that as a separate commit with rationale etc.